app/models/taxon.rb

```diff
@@ -2158,6 +2158,92 @@ def flagged_with( _flag, _options = {} )
     elastic_index!
   end
 
+  def clash_analysis( new_parent_id )
+    if INatAPIService.get( "/identifications", { current: true, exact_taxon_id: id, per_page: 0 } ).total_results.zero?
+      []
```

Comment:
Is this supposed to be `return []`?

Reply:
not according to rubocop

Reply:
I don't know what that means. As written this `if` statement does nothing, so why is it here?

Reply:
I guess you're right, and rubocop no longer complaining with that return. Thanks